src/components/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  ListboxInput,
+  ListboxButton,
+  ListboxPopover,
+  ListboxList,
+  ListboxOption,
+} from '@reach/listbox';
+import '@reach/listbox/styles.css';
+import styled from 'styled-components';
+
+const StyledListboxInput = styled(ListboxInput)`
+  cursor: pointer;
+`;
+const StyledListboxPopover = styled(ListboxPopover)`
+  z-index: 200;
+  cursor: pointer;
+  padding: 0;
+`;
+
+const StyledListboxButton = styled(ListboxButton)`
+  border: none;
+  cursor: pointer;
+`;
+
+interface DropdownProps {
```

Comment:
See #228: It should be possible to extend the react components interface and have all default `Sekect` props at once: 
```
interface test extends React.ComponentPropsWithoutRef <'select'> {}
```

Reply:
`React.ComponentPropsWithoutRef <'select'>` doesn't work, because the reach ui listbox uses other props/types than the native `select` element. 
Reach UI provides type definitions for some components, unfortunately I didn't find any for the listbox

Reply:
Maybe you can dig into the sources of the component by cloning the repo? Did you try that? It's maybe a mistake of not exporting something on their side ?